app/cli/cmd/integration_add.go

```diff
@@ -19,12 +19,15 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var integrationDisplayName string
+
 func newConfigIntegratioAddCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "add",
 		Short: "Add integration",
 	}
 
+	cmd.PersistentFlags().StringVar(&integrationDisplayName, "display-name", "", "integration registration description")
```

Comment:
This is added to the parent command so it can be reused by future integrations

Reply:
I am not sure about calling it a display name,this is a bit confusing. Why do we try to call it a displayName in the CLI and use for a integration instance description. Why not just name or description?

Reply:
My reasoning is as follows, and sorry I should have given this context before.

The `integration` model and DB entry describes an integration registration instance and in that context, the `description` term makes sense, it's a description of the registration entity.

Once we move to the CLI, to the `integration add/list` command, the `description` term can be ambiguous, is this the description of the registration or from the actual `extension`, shown as `kind` (which will likely have a description in the future). To make things worse I don't think we can call this `registration description` because the `registration` term has not been introduced in the UX.

So to summarize, description makes sense once you are in a specific context (db entry) but in the UX where both registration + extension information is shown it can be ambiguous.

That said, for consistency, and so we can tackle this in the future with a broader picture I'll rename this to description as well. 

Thanks!